tasks/src/main/scala/dagr/tasks/fgbio/FilterConsensusReads.scala

```diff
@@ -29,26 +29,38 @@ import dagr.tasks.DagrDef.PathToBam
 
 import scala.collection.mutable.ListBuffer
 
-class FilterConsensusReads(val in: PathToBam,
-                           val out: PathToBam,
-                           val ref: PathToFasta,
-                           val reverseTags: Boolean = false,
-                           val minReads: Int,
-                           val maxReadErrorRate: Double,
-                           val minQuality: Int,
-                           val maxBaseErrorRate: Double,
-                           val maxNoCallFraction: Double)
+object FilterConsensusReads {
+  def apply(in: PathToBam,
+            out: PathToBam,
+            ref: PathToFasta,
+            minReads: Int,
+            maxReadErrorRate: Double,
+            maxBaseErrorRate: Double,
+            minQuality: Int,
+            maxNoCallFraction: Double): FilterConsensusReads = {
+    apply(in=in, out=out, ref=ref, minReads=Seq(minReads), maxReadErrorRate=Seq(maxReadErrorRate),
+      maxBaseErrorRate=Seq(maxBaseErrorRate), minQuality=minQuality, maxNoCallFraction=maxNoCallFraction)
+  }
+}
+
+case class FilterConsensusReads(in: PathToBam,
```

Comment:
That and also because I wanted to have a pair of apply methods, one that took Seqs and one that took single values, because I think that's nicer than having two constructors, and this way I get one of them for free.